package.json

```
@@ -72,7 +72,7 @@
     }
   },
   "scripts": {
-    "bootstrap": "npm-run-all -s check-versions lerna-prepare",
+    "bootstrap": "npm-run-all -s check-versions \"lerna-prepare -- --{@}\" --",
```

We use yarn on CI so we shouldn't need lerna-prepare with -- as yarn works without -- to pass arguments down.
```suggestion
    "bootstrap": "npm-run-all -s check-versions lerna-prepare",
```

---

Already tried this. It doesn't work because both `check-versions` and `lerna-prepare` are scripts in `package.json`

---

My bad, i had a dependency error, not a lerna-prepare issue. This works on windows too! :) great job! Sorry about being faulty on this one! <3